### PR TITLE
Add periodic payments support in ing-adapter

### PR DESCRIPTION
**Pull request by artemsnisarenko**

### CODE REVIEW

File: adapters/ing-adapter/src/main/java/de/adorsys/xs2a/adapter/ing/IngMapper.java [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+import org.mapstruct.Mapping;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+@Mapper
+public interface IngMapper {
+    AccountList map(IngAccountsResponse value);
```
Comment (spetrychenko) [inline adapters/ing-adapter/src/main/java/de/adorsys/xs2a/adapter/ing/IngMapper.java:17]: Can't find teste for this mapper, does it exist?

Comment (artemsnisarenko) [reply adapters/ing-adapter/src/main/java/de/adorsys/xs2a/adapter/ing/IngMapper.java:17]: no

Comment (spetrychenko) [reply adapters/ing-adapter/src/main/java/de/adorsys/xs2a/adapter/ing/IngMapper.java:17]: please add it

Comment (artemsnisarenko) [reply adapters/ing-adapter/src/main/java/de/adorsys/xs2a/adapter/ing/IngMapper.java:17]: added

**Pull request merged**